mosfit/utils.py

```diff
@@ -318,8 +318,12 @@ def congrid(a, newdims, method='linear', center=False, minusone=False,
 
 def all_to_list(array):
     """Recursively convert list of numpy arrays to lists."""
-    return [x.tolist() if type(x).__module__ == 'numpy'
-            else all_to_list(x) if type(x) == 'list' else x for x in array]
+    try:
+        return ([x.tolist() if type(x).__module__ == 'numpy'
+                else all_to_list(x) if type(x) == 'list' else
+                x for x in array])
+    except TypeError:  # array is not iterable
+        return [array]
```

Comment:
I assumed that since it looked like this function was trying to always return lists previously (and is called all_to_list) that we'd want to still return a list. But in the specific case of returning extra outputs i'm not sure if it matters.